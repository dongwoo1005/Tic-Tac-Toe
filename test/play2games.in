game stdin stdin
NW
C
SW
N
W
game stdin stdin
NW
C
SW
N
W
quit
