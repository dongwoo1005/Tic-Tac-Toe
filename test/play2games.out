X's move
O's move
X's move
O's move
X's move
X wins
Score is
X 1
O 0
O's move
X's move
O's move
X's move
O's move
O wins
Score is
X 1
O 1
